test/e2e/resources/notebook_instance_lifecycle_config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: NotebookInstanceLifecycleConfig
+metadata:
+  name: $NOTEBOOK_INSTANCE_LFC_NAME
+spec:
+  notebookInstanceLifecycleConfigName: $NOTEBOOK_INSTANCE_LFC_NAME
+  onStart:
+    # base64 decoded string: A="5"
```

Comment:
oh this is what you meant in the samples 
can we have something like this instead `echo "Entering onStart"`

and `echo "Entering onCreate"`

Reply:
done